pymc_bart/utils.py

```diff
@@ -330,6 +331,9 @@ def plot_pdp(
         List of the indices of the covariate treated as discrete.
     func : Optional[Callable], by default None.
         Arbitrary function to apply to the predictions. Defaults to the identity function.
+    softmax_link: Optional[bool] = False,
```

Comment:
We already have a `func` argument. We should use that argument to pass arbitrary functions, including the softmax.

Reply:
Addressed @aloctavodia